benchopt/tests/dummy_benchmark/doc_objective.py

```diff
@@ -1,4 +1,4 @@
-from benchopt.base import BaseObjective, safe_import_context
+from benchopt import BaseObjective, safe_import_context
```

Comment:
```suggestion
from benchopt import safe_import_context
from benchopt.base import BaseObjective
```

Reply:
in fact it's not necessary. Probably was changed in latest dev version ? I don't think it used to work with 1.4.0.